profile/README.md

```diff
@@ -1,32 +1,43 @@
+data.
 
 <br clear="right"/>
 
-## Arcaflow
+## <a href="https://arcalot.io/arcaflow">Arcaflow</a>
 
-<img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png">
+<a href="https://arcalot.io/arcaflow"><img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png"></a>
 
-Arcaflow is a workflow engine consisting of three main components:
+Arcaflow is a workflow system consisting of three main components:
 
-* Core engine
-* UI
-* Plugins (including SDKs for Go and Python to write your own plugins)
+* A core engine, statically-compiled and platform-portable, written in Go
+* Plugins, written in various languages and delivered as Linux containers or modules
+* Workflow definitions, written in YAML, tieing together plugins and their data
 
-It allows you to click and drag plugins into a workflow for your systems and, if needed, feed the resulting data back into the UI for further analysis. You can also use it just to generate a workflow with parallel and subsequent tasks via the command line. There is a range of supported plugins, written either in Go or Python.
+It allows you create workflows with parallel and serial tasks, passing data between the
+tasks to achieve simple or complex pipelines of actions that result in machine-readable
+outputs. The community supports an ever-growing variety of plugins, and we provide SDKs
+in Go and Python to aid the user in developing their own custom plugins.
+
+Arcaflow allows you to encapsulate and version-control expertise, making potentially
+very complex workflows easily portable among environments and automation systems. With
+an Arcaflow workflow, you can carefully craft a pipeline of actions that serves your
+direct needs, and share that worflow virtually unchanged for others to run in different
+environments and CI/CD systems.
 
 <br clear="left"/>
 
-## Arcalog
+## <a href="https://arcalot.io/arcalog/">Arcalog</a>
 
-<img align="right" width="200px" style="padding-left: 2em;"  alt="Arcalog logo showing an ark floating on a blue scroll symbolizing the many logs it is scrolling through" src="https://github.com/arcalot/.github/raw/main/branding/arcalog.png">
+<a href="https://arcalot.io/arcalog/"><img align="right" width="200px" style="padding-left: 2em;"  alt="Arcalog logo showing an ark floating on a blue scroll symbolizing the many logs it is scrolling through" src="https://github.com/arcalot/.github/raw/main/branding/arcalog.png"></a>
 
 Arcalog can assist you with or automate your root cause analysis in CI or other log systems either as a standalone tool or by embedding it into your applications.
```

Comment:
Should we be removing this section, or is that out of scope for this PR?

Reply:
It's been removed now.